src/anjaliP/WhileLoop.java

```diff
@@ -0,0 +1,47 @@
+package anjaliP;
+
+public class WhileLoop {
+	void display1(int start,int end) {
+		System.out.println("-------------Even Numbers----------------");
+		int index=start;
```

Comment:
You can handle this scenario without creating third variable, think on this.
can you use startNumber++ for increment?